http-server-netty/src/test/groovy/io/micronaut/http/server/netty/cors/CorsVersionSpec.groovy

```diff
@@ -113,7 +112,7 @@ class CorsVersionSpec extends Specification {
 
         when:
         request = HttpRequest.OPTIONS("/new-not-allowed-from-private")
-        preflightHeaders(null, true).each { k, v -> request.header(k, v)}
+        preflightHeaders("x-api-version", true).each { k, v -> request.header(k, v)}
```

Comment:
what the reason of "x-api-version" here?

Reply:
While debugging, I discovered that without the "Access-Control-Request-Headers" header, the test failed to target the correct `TestController` endpoint (annotated with `@CrossOrigin(allowPrivateNetwork = false)` and `@Version("2")`).

Before the proposed changes, I believe the test passed for the wrong reason — the `403 FORBIDDEN` response occurred because the route was not found, not due to the `@CrossOrigin(allowPrivateNetwork = false)` configuration.

I think the issue is related to the `@Version("2")` annotation since temporarily removing this annotation resolved the problem, but I then decided not to remove the annotation and add the header instead.

Reply:
ive seen to the code. this behaviour happens by the reason that matchesOrigin method at CorsFilter returns false if we defined Version annotation but request will not support this header cause accessControlRequestHeaders is null and then it responce 200 cause all cors requests return 200 by this pr. so i guess it test right, thanks